__tests__/generate.test.js

```diff
@@ -5,7 +5,7 @@ const { join } = require('path')
 describe('generate', () => {
     it('should create a new migration file', async () => {
         const migrationName = 'test-migration'
-        const { stdout } = await execa('cmp', ['generate', migrationName])
+        const { stdout } = await execa.node('./bin/cmp.js', ['generate', migrationName])
```

Comment:
For me, just `execa('./bin/cmp.js', ['generate', migrationName])` works also, don't know why.